core/persistence-jpa/src/test/resources/domains/MasterContent.xml

```diff
@@ -461,7 +470,15 @@ under the License.
   <SyncopeSchema id="mderToBePropagated"/>
   <DerSchema id="mderToBePropagated" expression="mderived_sx + '-' + mderived_dx" 
              anyTypeClass_id="generic membership"/>
-        
+
+  <SyncopeSchema id="lastRecertification"/>
```

Comment:
We should probably better handle `lastRecertification` and `lastRecertificator` rather as `User` / `JPAUser` fields rather than schemas.